src/ansys/dpf/composites/_composite_model_impl.py

```diff
@@ -144,6 +144,32 @@ def material_operators(self) -> MaterialOperators:
         """Material operators."""
         return self._material_operators
 
+    @property
+    def material_names(self) -> Dict[str, int]:
+        """
+        Material name to DPF material ID map.
+
+        Can be used to filter analysis plies or element layers.
```

Comment:
While the short description is a sentence fragment, any additional information should be in the form of a complete sentence: "This property can be used ...."